metrics-server/config/rbac/role.yaml

```diff
@@ -6,6 +6,61 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
RBAC change looks good -- operator needs write access in namespace for objects within channel manifests